openapi_python_client/parser/properties/property.py

```diff
@@ -31,21 +31,29 @@ class Property:
 
     def __attrs_post_init__(self) -> None:
         object.__setattr__(self, "python_name", utils.to_valid_python_identifier(utils.snake_case(self.name)))
+    
+    def get_base_type_string(self) -> str:
+        return self._type_string
 
-    def get_type_string(self, no_optional: bool = False) -> str:
+    def get_type_string(self, no_optional: bool = False, query_parameter: bool = False) -> str:
```

Comment:
Would it make more sense to label this flag more to what it does than how it's used along the lines of
```suggestion
    def get_type_string(self, no_optional: bool = False, treat_none_as_unset: bool = False) -> str:
```
(I don't think I really like `treat_none_as_unset`, but you get my point)

Reply:
Hmm, interesting idea. I could see that being useful in a future world where `treat_none_as_unset` might be used in different situations. Since for now we only have one use case, which is query parameters, I think it's more descriptive to keep it as is (especially if we end up changing the behavior of query parameters again).